tests/test_features/test_pca.py

```diff
@@ -176,7 +176,7 @@ def test_biplot_3d(self):
         Test the PCADecomposition 3D biplot (proj_features).
         """
         params = {
-            "features": "ABCDEFGHIKLM",
+            "features": list("ABCDEFGHIKLM"),
```

Comment:
When `self.features_` was changed to a `np.array` you can no longer treat a string like a list.

Reply:
:+1: